tests/Functional/app/config/services_test.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+
+  Silverback\ApiComponentBundle\Features\Bootstrap\:
+    resource: '../../../../features/bootstrap/*'
+
+  Silverback\ApiComponentBundle\Features\Bootstrap\DoctrineContext:
+    arguments:
+      $jwtManager: '@lexik_jwt_authentication.jwt_manager'
```

Comment:
Using [`JWTTokenManagerInterface`](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Services/JWTTokenManagerInterface.php) for your constructor argument type (and not `JWTManagerInterface` which is deprecated) would make it autowired, which would avoid the need for defining this service explicitly

Reply:
Thanks !